### PR TITLE
New package: glaxnimate-0.5.1

### PR DESCRIPTION
**Pull request by classabbyamp**
- New package: python3-lottie-0.7.0
- New package: glaxnimate-0.5.1
- kdenlive: add missing dependencies

#### Testing the changes
- I tested the changes in this PR: **YES**|**briefly**|**NO**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

### CODE REVIEW

File: srcpkgs/kdenlive/template [hunk 1]
```diff
@@ -10,7 +10,8 @@ makedepends="
  kdeclarative-devel kfilemetadata5-devel knewstuff-devel knotifyconfig-devel
  kplotting-devel mlt7-devel qt5-multimedia-devel qt5-webkit-devel purpose-devel
  v4l-utils-devel ksolid-devel qt5-quickcontrols2-devel qt5-networkauth-devel"
-depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit qt5-quickcontrols"
+depends="breeze-icons dvdauthor ffmpeg frei0r-plugins kinit qt5-quickcontrols
+ kirigami2 mediainfo"
```
Comment (JamiKettunen) [inline srcpkgs/kdenlive/template:14]: Appears `qt5-graphicaleffects` might be missing too as per:
```
$ kdenlive
...
qrc:/qml/timeline.qml:1177:25: Type TrackHead unavailable 
                             TrackHead { 
                             ^
qrc:/qml/TrackHead.qml:10:1: module "QtGraphicalEffects" is not installed 
     import QtGraphicalEffects 1.15 
     ^
QObject::connect: Cannot connect (nullptr)::mousePosChanged(int) to MainWindow::slotUpdateMousePosition(int)
QObject::connect: Cannot connect (nullptr)::zoomIn(bool) to MainWindow::slotZoomIn(bool)
QObject::connect: Cannot connect (nullptr)::zoomOut(bool) to MainWindow::slotZoomOut(bool)
QObject::connect: Cannot connect (nullptr)::processingDrag(bool) to MainWindow::enableUndo(bool)
Segmentation fault
```
(full launch log https://termbin.com/ex3i)

**Comment by classabbyamp** [comment]
based on https://github.com/mltframework/mlt/issues/857 and https://gitlab.com/mattbas/glaxnimate/-/issues/545, it's not possible to get glaxnimate working with kdenlive yet without building it twice (yuck). however, the "missing component" message should be nonfatal.

I think I will keep this as a PR until it gets some more testing, but merge the kdenlive fix.

File: srcpkgs/kdenlive/template (hunk 1 above)
Comment (classabbyamp) [reply srcpkgs/kdenlive/template:14]: hm i have that installed but I still have the segfault

**Comment by Andonome** [comment]
Is there any word on this?

Kdenlive crashes on startup for me, but after pulling this branch, and dealing with the conflict, I was able to compile a working kdenlive.

I don't want to verify the changes personally, as I don't properly understand xbps-src, but `kdenlive` is certainly working for me (but not getting any updates).

**Pull request closed without merging**